app/assets/stylesheets/components/_memorable-date.scss

```
@@ -0,0 +1,4 @@
+.input-padding {
```

This is a pretty generic name that might imply that it's reusable outside the component. I would expect any selectors in the component's stylesheet to be named using [BEM style](https://en.bem.info/methodology/naming-convention/#two-dashes-style).

```suggestion
.memorable-date {
```

---

I went for `.usa-memorable-date input[type='text']` as the selector.

---

> I went for `.usa-memorable-date input[type='text']` as the selector.

In retrospect, I think that's actually a much better option, since after my review I was wondering why we couldn't just rely on the component styles from the design system. I think eventually we may be able to remove these altogether, once we get rid of the "design system waiting room" styles, since it looks like the USWDS default input padding is already `0.5rem`.